### PR TITLE
loops / it-academy.by / M-JC1-66-20 / Dunin Eugene / Done tasks 1 - 10

### PR DESCRIPTION
**Pull request by Eugenoil**
## Pull request state

Please indicate actual status for PR:
- [x] on review
- [ ] reviewed
- [ ] fixed
- [ ] on approve
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/javabasics/practice/loops/Task02.java [hunk 1]
```diff
@@ -34,6 +34,19 @@ public static void main(String[] args) {
     static String generateNumbersColumn(int numberRows, boolean isEnableHeaderRow) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        StringBuilder table_1 = new StringBuilder();
+        StringBuilder table_2 = new StringBuilder();
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/javabasics/practice/loops/Task02.java:38]: Why are you using `StringBuilder`?)

File: src/main/java/com/rakovets/course/javabasics/practice/loops/Task06.java [hunk 2]
```diff
@@ -36,6 +36,15 @@ public static void main(String[] args) {
     static String convertToAccountingFormat(long amount) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int length = String.valueOf(amount).length();
+        StringBuilder sum = new StringBuilder((Long.toString(amount)));
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/javabasics/practice/loops/Task06.java:40]: Why are you using `StringBuilder`?)

**Review by rakovets — COMMENTED** [review]
Good job but Why are you using `StringBuilder`?)

File: src/main/java/com/rakovets/course/javabasics/practice/loops/Task01.java [hunk 3]
```diff
@@ -31,6 +31,11 @@ public static void main(String[] args) {
     static String generateNumbersColumn(int numberRows) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        StringBuilder table = new StringBuilder();
```
[the remark this answers is not in the document]
Comment (Eugenoil) [reply src/main/java/com/rakovets/course/javabasics/practice/loops/Task01.java:34]: Because IDEA suggested using StringBuilder instead of my expressions)

**Pull request closed without merging**